### ISSUE TITLE
attention权重

### ISSUE THREAD

**Issue by flowerflow** [NLPxiaoxu/Easy_Rnn_Attention#1]
您好，谢谢您的project，让我对attention机制有了更多的理解，有两个个问题想请教一下您，希望您能帮助我一下。
1：在训练完后，我如何查看一个词向量的权重呢？
2：我能否在训练的中期，对指定的词汇，修改他的权重呢？
还望您解惑。thanks

**Comment by X-jun-0130** [comment]
weights = tf.nn.softmax(logit, name='attention_weights') 这个值就是权重，您可以在最后的session里面把值输出来。对于您，第二个问题，我仔细想了想，也上网查了下。我想不出来有什么好的方法修改权重。神经网络在训练时，跟传统机器学习算法的区别还是很明显的，神经网络是个黑匣子的操作，中间操作不给你看，直接从输入得到输出的结果。

**Comment by flowerflow** [comment]
很高兴您能回复我（虽然过去了很久，我都已经搞懂了你代码的大部分了hhhh，而且说‘您‘，感觉好隆重啊2333
不过还是很感谢你的project，学习到了很多，thanks！  =V=

**Comment by X-jun-0130** [comment]
哈哈 我之前在找工作 、搬家  所以很长时间没上来。很高兴能够帮助你

**Comment by flowerflow** [comment]
年入百万的大佬加油~~
